### PR TITLE
Modular bjtu_homework_tracker

### PR DESCRIPTION
**Pull request by ymzhang-cs**
Only contains necessary codes for development.

It has been tested and able to setup with these command:

```bash
pip install -e .
```

```bash
PS > pip install -e .                
...
Successfully installed bjtu_homework_tracker-0.2.0
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,5 +4,4 @@ ref/*
 
 README_RAW.md
 history.txt
-config.yaml
 search_history/*
```
Comment (ram6ert) [inline .gitignore:7]: 为什么取消掉ignore了

File: bjtu_homework_tracker/config_sample.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ login:
   mis:
     # Path to the webdriver (leave empty if you want to use the default path)
     webdriver_path: ""
-    browser: edge
+    browser: chrome
```
Comment (ram6ert) [inline bjtu_homework_tracker/config_sample.yaml:17]: 怎么改成chrome了

**Review by ram6ert — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (ymzhang-cs) [reply .gitignore:7]: 因为api里面有一个需要 `config.yaml` 占位的文件，所以临时取消。之后应该还会再加回来~

File: bjtu_homework_tracker/config_sample.yaml (hunk 2 above)
Comment (ymzhang-cs) [reply bjtu_homework_tracker/config_sample.yaml:17]: 因为我用 Chrome🤣

**Pull request merged**